modules/administration-guide/pages/configuring-autoscaling.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: ASSEMBLY
+:description: Configuring autoscaling
+:keywords: administration-guide, configuring, autoscaling, autoscale, horizontal, scaling, machine
+:navtitle: Configuring autoscaling
+:page-aliases:
+
+[id="configuring-autoscaling"]
+= Configuring autoscaling
+
+This section describes different aspects of autoscaling for {prod}.
```

Comment:
which docs do you want to link here? configuring machine autoscaling and configuring number of replicas?

Reply:
Yes, I've added them in my latest commit